csvexport/src/csvexport.cpp

```diff
@@ -23,12 +23,13 @@ void print_usage_exit(int e)
     fprintf(stderr, "Usage:\n");
     fprintf(stderr, "  extract [OPTION...] <trace file>\n");
     fprintf(stderr, "\n");
-    fprintf(stderr, "  -h, --help        Print usage\n");
-    fprintf(stderr, "  -f, --filter arg  Filter zone names (default: "")\n");
-    fprintf(stderr, "  -s, --sep arg     CSV separator (default: ,)\n");
-    fprintf(stderr, "  -c, --case        Case sensitive filtering\n");
-    fprintf(stderr, "  -e, --self        Get self times\n");
-    fprintf(stderr, "  -u, --unwrap      Report each zone event\n");
+    fprintf(stderr, "  -h, --help                       Print usage\n");
+    fprintf(stderr, "  -f, --filter arg                 Filter zone names (default: "")\n");
+    fprintf(stderr, "  -s, --sep arg                    CSV separator (default: ,)\n");
+    fprintf(stderr, "  -c, --case                       Case sensitive filtering\n");
+    fprintf(stderr, "  -e, --self                       Get self times\n");
+    fprintf(stderr, "  -u, --unwrap                     Report each zone event\n");
+    fprintf(stderr, "  -m, --unwrap only messages       Report only messages\n");
```

Comment:
This does not correspond with what the program expects.

Reply:
Hi @wolfpld there is currently no way to print messages which we add to zones.

Reply:
@wolfpld would this be more acceptable?
```suggestion
    fprintf(stderr, "  -m, --messages                   Report only messages\n");
```

Reply:
Yes, but the additional padding added between arguments and their descriptions is no longer needed.